libs/StatusGoQt/src/StatusGo/Accounts/Accounts.cpp

```diff
@@ -0,0 +1,223 @@
+#include "Accounts.h"
+
+#include "Utils.h"
+#include "libstatus.h"
+
+const int NUMBER_OF_ADDRESSES_TO_GENERATE = 5;
+const int MNEMONIC_PHRASE_LENGTH = 12;
+
+namespace Status::StatusGo::Accounts {
+
+RpcResponse<QJsonArray> generateAddresses(const QVector<QString>& paths)
```

Comment:
Could we remove the json as early as possible from the call stack => ```RpcResponse<[]GeneratedAddress>```

If we call this method from more than one place, we have to redo the parsing which is quite error prone

This is a comment for all call to status-go

Reply:
Totally agree, that is the way to go for the module interface. However, I was not sure if we should do it in the thin wrapper or add another level to the data model. I have to understand the main context for this. Here I just migrated existing code to test status-go integration. Will come back to it soon, thanks for the heads up.